code/datums/diseases/advance/advance.dm

```diff
@@ -33,7 +33,7 @@
 	var/mutable = TRUE //set to FALSE to prevent most in-game methods of altering the disease via virology
 	var/oldres
 
-	// The order goes from easy to cure to hard to cure.

```

Comment:
Needs Documentation regarding the fact SD doesn't use 1-5

Reply:
Fixed